IBAnimatable/Utils.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Utils.swift
+//  IBAnimatableApp
+//
+//  Created by jason akakpo on 03/07/16.
+//  Copyright © 2016 Jake Lin. All rights reserved.
+//
+
+import Foundation
+
+internal extension Array {
+  /// Returns the element at the specified index iff it is within bounds, otherwise nil.
+  subscript(safe index: Int ) -> Element? {
+    return indices.contains(index) ? self[index] : nil  /// Returns the element at the specified index iff it is within bounds, otherwise nil.
```

Comment:
Typo: `iff` -> `if`
Since it's a duplication of the same comment, is it really useful?